신은지/Day1_hash.py

```diff
@@ -0,0 +1,14 @@
+def solution(participant, completion):
+    # 효율성 실패
+    # 해시 사용하지 않음
+    # for i in completion:
+    #     participant.remove(i)
+    # answer = participant[0]
+    # return answer
+
+    participant.sort()
+    completion.sort()
+
+    for i in range(0,len(completion)):
```

Comment:
인덱스가 0부터 시작한다면 마지막 조건만 적어도 됩니다.
```suggestion
    for i in range(len(completion)):
```